modules/dbfs/mysql.go

```diff
@@ -0,0 +1,452 @@
+package dbfs
+
+import (
+	"database/sql"
+
+	_ "github.com/go-sql-driver/mysql" // required to load into local namespace to
+	// initialize sql driver mapping in sql.Open("mysql", ...)
+	"strconv"
+
+	"time"
+
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/CodeCollaborate/Server/modules/config"
+	"github.com/CodeCollaborate/Server/utils"
+)
+
+var mysqldb *mysqlConn
+
+var mysqldbName = "cc"
+
+type mysqlConn struct {
+	config config.ConnCfg
+	db     *sql.DB
+	dbname string
+}
+
+func openMySQLConn(dbName string) (*mysqlConn, error) {
+	if mysqldb != nil && mysqldb.db != nil {
+		err := mysqldb.db.Ping()
+		if err == nil && mysqldb.dbname == dbName {
+			return mysqldb, nil
+		}
+	}
+
+	if mysqldb == nil || mysqldb.config == (config.ConnCfg{}) {
+		mysqldb = new(mysqlConn)
+		configMap := config.GetConfig()
+		mysqldb.config = configMap.ConnectionConfig["MySQL"]
+	}
+
+	db, err := sql.Open("mysql", mysqldb.config.Username+":"+mysqldb.config.Password+"@tcp("+mysqldb.config.Host+":"+strconv.Itoa(int(mysqldb.config.Port))+")/"+dbName+"?timeout="+strconv.Itoa(int(mysqldb.config.Timeout))+"s"+"&parseTime=true")
```

Comment:
perhaps use a formatted string? it's a little hard to read what the different parts of the connection string are